### DISCUSSION TITLE
tutorials/shaders/making_trees

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#185] [User-contributed notes]
# tutorials/shaders/making_trees

This is a short tutorial on how to make trees and other types of vegetation from scratch. The aim is to not focus on the modeling techniques (there are plenty of tutorials about that), but how to m...

https://docs.godotengine.org/en/stable/tutorials/shaders/making_trees.html

**Comment by heinermann** [comment]
For anyone who has created trees using TreeIt (which automatically generates vertex colours described above), someone created a Godot-ready shader over at https://godotshaders.com/shader/treeit-tree-shader/

**Reply by ianmckenna87** [reply]
could do with some more instructions on that workflow. Can't use both the tree material and that shader at the same time
